src/systems/systems/src/engine/mod.rs

```diff
@@ -4,7 +4,7 @@ use crate::simulation::{SimulationElement, SimulatorReader, UpdateContext};
 
 pub struct Engine {
     corrected_n2_id: String,
-    corrected_n2: Ratio,
+    pub corrected_n2: Ratio,
```

Comment:
**issue**: This field should no longer be public.
**suggestion**: I remember this being a remnant of not yet having the `SimulationTestBed` available. Now that it is available, you should be able to make this private again and instead set the appropriate simulation variable.

Reply:
Yup! think I should be using the corrected_n2(&self) getter now.